zkevm-circuits/src/util/word.rs

```diff
@@ -151,20 +259,46 @@ impl<T> std::ops::Deref for Word<T> {
     }
 }
 
+impl<T: Clone + PartialEq> PartialEq for Word<T> {
+    fn eq(&self, other: &Self) -> bool {
+        self.lo() == other.lo() && self.hi() == other.hi()
+    }
+}
+
 impl<F: Field> Word<F> {
-    /// Constrct the word from u256
+    /// Construct the word from u256
```

Comment:
Maybe we could use the `From` trait for all these constructors?

Reply:
nice call and thanks!, i have addressed it in https://github.com/privacy-scaling-explorations/zkevm-circuits/pull/1441/commits/79025d72da404f0b9b0d74a0ab3f7b97881acc7c